test/get_previous_releases.py

```diff
@@ -148,6 +148,24 @@ def download_binary(tag, args) -> int:
     ret = subprocess.run(['tar', '-zxf', tarball, '-C', tag,
                           '--strip-components=1',
                           'bitcoin-{tag}'.format(tag=tag[1:])]).returncode
+    
+    if tag >= "v23" and platform == "arm64-apple-darwin":
+        # Starting with v23.0 there are arm64 binaries for ARM (M1/M2) macs, but they have to be signed to run
+        binary_path = '{cwd}/{tag}/bin/bitcoind'.format(cwd=os.getcwd(), tag=tag)
```

Comment:
For new code f-strings are recommended:

> Use `f'{x}'` for string formatting in preference to `'{}'.format(x)` or `'%s' % x`.

See:
- https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-python
- https://github.com/bitcoin/bitcoin/blob/master/test/functional/README.md#style-guidelines